### ISSUE TITLE
python-eccodes is not installable

### ISSUE THREAD

**Issue by zklaus** [EC-Earth/ece2cmor3#729]
In the middle of 2019, ECMWF split python-eccodes from eccodes. This separation is good, but at the same time they reset the version numbering for python-eccodes. The last version of the old scheme, where both eccodes and its python binding had the same version number, was 2.12.3. The first version number in the new scheme was 0.9.2. In order to avoid the larger version numbers overriding the newer versions with smaller numbers, the older versions were marked as broken. The newer builds, however, are only provided for Python 3 since Python 2 has been dead for over two years (see https://endoflife.date/python). Because ece2cmor is unfortunately still Python 2, this means that currently new installations with the existing `environment.yml` file are not possible.

**Comment by treerink** [comment]
#403 related

**Comment by zklaus** [comment]
@treerink, yes, quite right. I have a short-term solution that I am preparing right now. Stay tuned...
